### ISSUE TITLE
Ping / pong events is not working.

### ISSUE THREAD

**Issue by matheust3** [uNetworking/uWebSockets#960]
I am using dart websockets:

**Dart:**
```
  bool connect() {
    _channel = IOWebSocketChannel.connect("ws://$_serverHost:$_serverPort/",
        headers: {"user": "any"}, pingInterval: Duration(seconds: 6) );
    _channel.sink.add("connected!");
    _channel.stream.listen((onData){
      print(onData);
    });
    return true;
  }
```
**C++:**
```
void Server::Start() {
  /* ws->getUserData returns one of these */
  struct PerSocketData {
    std::string *userName;
  };
  /* Simple echo websocket server, using multiple threads */
  std::vector<std::thread *> threads(std::thread::hardware_concurrency());

  std::transform(
      threads.begin(), threads.end(), threads.begin(),
      [](std::thread *t) {
        return new std::thread([]() {
          /* Very simple WebSocket echo server */
          uWS::App()
              .ws<PerSocketData>(
                  "/*", {/* Settings */
                         .compression = uWS::SHARED_COMPRESSOR,
                         .maxPayloadLength = 16 * 1024,
                         .idleTimeout = 10,
                         .maxBackpressure = 1 * 1024 * 1204,
                         /* Handlers */
                         .open =
                             [](auto *ws, auto *req) {
                               PerSocketData *data = (PerSocketData *)ws->getUserData();
                               //data = new PerSocketData;
                               data->userName = new std::string(req->getHeader("user"));
                               std::cout << *data->userName <<" se conectou!" << std::endl;
                             },
                         .message =
                             [](auto *ws, std::string_view message,
                                uWS::OpCode opCode) {
                               std::cout << message << std::endl;
                             },
                         .drain =
                             [](auto *ws) {
                               // Check getBufferedAmount here
                               std::cout << "drain" << std::endl;
                             },
                         .ping =
                             [](auto *ws) {
                               std::cout << "ping" << std::endl;
                             },
                         .pong =
                             [](auto *ws) {
                               //
                               std::cout << "pong" << std::endl;

                             },
                         .close =
                             [](auto *ws, int code, std::string_view message) {
                               //
                               PerSocketData *data = (PerSocketData *)ws->getUserData();
                               std::cout << *data->userName << " desconectado!" << std::endl;
                              delete data->userName;
                             }})
              .listen(9001,
                      [](auto *token) {
                        if (token) {
                          std::cout << "Thread " << std::this_thread::get_id()
                                    << " listening on port " << 9001
                                    << std::endl;
                        } else {
                          std::cout << "Thread " << std::this_thread::get_id()
                                    << " failed to listen on port 9001"
                                    << std::endl;
                        }
                      })
              .run();
        });
      });

  std::for_each(threads.begin(), threads.end(),
                [](std::thread *t) { t->join(); });
}
```

but ping / pong is not working correctly. When the client sends ping the event is not called on the server;

**Comment by alexhultman** [comment]
Ping pong events are not delivered to the application, that should be fixed at some time

**Comment by alexhultman** [comment]
It should work now, you won't get the ping (payload) message though, as that would break current API. Will fix for v0.18
